src/components/Pages/Search/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect  } from 'react'
+import { useRouteData, Head } from 'react-static';
+import { useIntl } from 'react-intl';
+import { events as i18n } from 'js/i18n/definitions';
+
+import PageHeader from 'components/PageHeader';
+import SearchResult from 'components/Pages/Search/searchResult.js'
+
+/* - - - TO DO: Finish Janis Search Page
+  - Issue: https://github.com/cityofaustin/techstack/issues/4285
+  // import { searchWorker } from 'js/helpers/searchWorker'
+  // import { loader } from 'js/animations/loader';
+*/
+
+const SearchPage = () => {
+
+  const { searchIndex } = useRouteData();
+
+  const title = "Search" // TODDO: ⚠️useIntl
```

Comment:
I approve of TODDO as a todo alternative

![image](https://user-images.githubusercontent.com/6384156/81437390-b509d680-9130-11ea-812a-446aeb234f93.png)

Reply:
🙌